### PR TITLE
Tweaks on relogin screen

### PR DESCRIPTION
**Pull request by cecileboucheron**
* Left-aligns user dropdown on desktop and mobile
* Brings new input to password field on mobile (@MarcoPolo has to check on Android)
* Better "Create an account" button and placement
* blueGrey background
image
image
image

cc @adamjspooner

### CODE REVIEW

**Comment by adamjspooner** [comment]
Looks good to me on Android!
Screen Shot 2019-10-24 at 07 50 47

File: shared/common-adapters/plain-input.d.ts [hunk 1]
```diff
@@ -83,6 +83,7 @@ export type Props = {
   placeholderColor?: string
   rowsMin?: number
   rowsMax?: number
+  secureTextEntry?: boolean
```
Comment (buoyad) [inline shared/common-adapters/plain-input.d.ts]: I don't think this is controlling anything, add a `|| this.props.secureTextEntry` here?

https://github.com/keybase/client/blob/08eec9bc220d2348330b8756f55fbd4d11be4bc5/shared/common-adapters/plain-input.native.tsx#L231

Comment (cecileboucheron) [reply shared/common-adapters/plain-input.d.ts]: added!

File: shared/common-adapters/user-card.native.tsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ const styles = Styles.styleSheetCreate(() => ({
   inside: {
     ...Styles.globalStyles.flexBoxColumn,
     alignItems: 'stretch',
-    backgroundColor: Styles.globalColors.fastBlank,
+    backgroundColor: Styles.globalColors.blueGrey,
```
Comment (buoyad) [inline shared/common-adapters/user-card.native.tsx]: Is this needed? Other places this is used has a blue background from the container; I think this'd be unexpected to have on this component.

Comment (cecileboucheron) [reply shared/common-adapters/user-card.native.tsx]: So you'd just ditch `backgroundColor`?

Comment (buoyad) [reply shared/common-adapters/user-card.native.tsx]: Yeah

**Pull request merged**